src/main/java/frc/robot/constants/RobotComponents.java

```diff
@@ -99,11 +98,14 @@
 
     protected static class ShooterComponents {
         // TODO: Set correct MotorConfig
-        private static final MotorConfig MOTOR_CONFIG = new MotorConfig()
-                .withOpenLoopRampRate(3).withClosedLoopRampRate(3);
-        public static final TrigonTalonFX RIGHT_MOTOR = new TrigonTalonFX(CAN.Shooter.RIGHT_MOTOR_ID, MOTOR_CONFIG);
-        public static final TrigonTalonFX LEFT_MOTOR = new TrigonTalonFX(
-                CAN.Shooter.LEFT_MOTOR_ID, new MotorConfig(MOTOR_CONFIG).inverted(true));
+        public static final PIDFTalonFX RIGHT_MOTOR = new PIDFTalonFX(
+                CAN.Shooter.RIGHT_MOTOR_ID, new MotorConfig()
+                .withPID(RobotConstants.LOCAL_CONSTANTS.localShooterConstants.pidfCoefs)
+                .withOpenLoopRampRate(3).withClosedLoopRampRate(3));
+        public static final PIDFTalonFX LEFT_MOTOR = new PIDFTalonFX(
+                CAN.Shooter.LEFT_MOTOR_ID, new MotorConfig()
+                .withPID(RobotConstants.LOCAL_CONSTANTS.localShooterConstants.pidfCoefs)
+                .withOpenLoopRampRate(3).withClosedLoopRampRate(3).inverted(true));
```

Comment:
1. remove inverted.  2. lower ramp rae

Reply:
low the ramp rate to 2? in both open and close loop?